### DISCUSSION TITLE
OneDrive连接、鉴权异常

### DISCUSSION THREAD

**Discussion by ptpt-y** [remotely-save/remotely-save#755] [Q&A] [answered]
之前是能正常同步的，今早发现同步异常说连不上OneDrive，就试了一下重新鉴权，但OneDrive鉴权一直卡在“Connecting to OneDrive... Please DO NOT close this modal.”
请问该如何解决呢？
![image](https://github.com/user-attachments/assets/88eaaa79-59a5-44d5-b027-62ca07b8b313)

**Comment by fyears** [comment, marked as answer]
换个 vpn？

或者有别的设备的 rs 已经连接了的话，可以导入那个设备的连接信息

**Reply by ptpt-y** [reply]
谢谢，换了个节点解决了。
